kore/test/Test/Kore/Builtin/String.hs

```diff
@@ -35,31 +32,18 @@ import qualified Hedgehog.Gen as Gen
 import qualified Hedgehog.Range as Range
 import qualified Kore.Builtin.Builtin as Builtin
 import qualified Kore.Builtin.String as String
+import qualified Kore.Builtin.String.String as String
```

Comment:
Learned something new today. I didn't know that Haskell lets us reuse a qualifier. Just for my understanding: Are all exports from `Kore.Builtin.String` and `Kore.Builtin.String.String` made available under the `String` qualifier? What if there is a naming collision? Will GHC complain, or will one definition shadow the other?

Reply:
Yes, in case of a naming collision, GHC will give an `Ambiguous occurrence` error, but only when the ambiguous function name is actually used.

Reply:
I'm not sure, I think I remember GHC complaining in such a situation.